### PR TITLE
Fix javadoc

### PR DESCRIPTION
**Pull request by dalinaum**
@realm/java

### CODE REVIEW

File: realm/realm-annotations-processor/src/main/java/io/realm/processor/ModuleMetaData.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public boolean generate(ProcessingEnvironment processingEnv) {
         return true;
     }
 
-    // Detour needed to access the class elements in the array
+    // Detours needed to access the class elements in the array
```
Comment (cmelchior) [inline realm/realm-annotations-processor/src/main/java/io/realm/processor/ModuleMetaData.java]: Detour is correct here as there is only 1

**Comment by cmelchior** [comment]
Apart from small comment :+1:

**Comment by dalinaum** [comment]
Done.

**Comment by zaki50** [comment]
:+1:

**Pull request merged**